src/server/scripts/Northrend/Ulduar/HallsOfStone/instance_halls_of_stone.cpp

```diff
@@ -87,6 +93,11 @@ class instance_halls_of_stone : public InstanceMapScript
                 case GO_TRIBUNAL_CONSOLE: 
                     goTribunalConsoleGUID = go->GetGUID(); 
                     break;
+                case GO_TRIBUNAL_ACCESS_DOOR:
+                    goTribunalDoorGUID = go->GetGUID();
+                    go->SetGoState(GO_STATE_READY);
+                    go->SetGoState(GO_STATE_READY);
```

Comment:
doubled

Reply:
I mustsh have beensh drunk, Cheers... Yuck! 🍻

Reply:
not resolved yet